modules/querying-the-status-of-cluster-nodes-using-the-cli.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// *installing/validating-an-installation.adoc
+
+[id="querying-the-status-of-cluster-nodes-using-the-cli_{context}"]
+= Querying the status of the cluster nodes by using the CLI
+
+You can verify the status of the cluster nodes after an installation.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. List the status of the cluster nodes. Verify that the output lists all of the expected control plane and compute nodes and that each node has a `Ready` status:
++
+[source,terminal]
+----
+$ oc get nodes
+----
++
+.Example output
+[source,terminal]
+----
+NAME                        STATUS   ROLES    AGE   VERSION
+ip-<ip_address>.internal    Ready    worker   33m   v1.19.0+9f84db3
```

Comment:
I don't know that the node name typically has this format. I usually see the node name as having the hostname of the machine.

Reply:
@staebler thank you for highlighting this. Some provider-formatted host names include IPv4 addresses. Since the host names in this assembly do not need to be provider-specific, I have updated the examples with more generic names that do not include IP addresses.